pkg/apis/admissionregistration/v1beta1/defaults_test.go

```diff
@@ -101,6 +102,29 @@ func TestDefaultAdmissionWebhook(t *testing.T) {
 				}},
 			},
 		},
+		{
```

Comment:
this looks like a duplicate of the test case above, am I missing a difference?

Reply:
Isn't it a test to verify `SetDefaults_ServiceReference` in default?

Reply:
yeah, the scope=* case just got duplicated in a rebase. looks good now.